docs/sources/shared/flow/reference/components/http-client-proxy-config-description.md

```diff
@@ -0,0 +1,19 @@
+---
+aliases:
+- /docs/agent/shared/flow/reference/components/http-client-proxy-config-description-args/
+- /docs/grafana-cloud/agent/shared/flow/reference/components/http-client-proxy-config-description-args/
+- /docs/grafana-cloud/monitor-infrastructure/agent/shared/flow/reference/components/http-client-proxy-config-description-args/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/shared/flow/reference/components/http-client-proxy-config-description-args/
+- /docs/grafana-cloud/send-data/agent/shared/flow/reference/components/http-client-proxy-config-description-args/
+canonical: https://grafana.com/docs/agent/latest/shared/flow/reference/components/http-client-proxy-config-description-args/
+description: Shared content, http client config description
+headless: true
+---
+
+`no_proxy` can contain IPs, CIDR notation and domain names. IP and domain names can contain port numbers.
+If `no_proxy` is configured, `proxy_url` must be configured.
```

Comment:
It feels a little funny for us to describe something called `no_proxy` and then say "if you configure `no_proxy`, you must configure `proxy_url`." Maybe flipping the order of these would be useful? 

IMO, the way to describe `no_proxy` is as a denylist of IPs/CIDR notation/domain names where the proxy server _will not_ be used, and a connection is instead made directly (which is why you have to specify the `proxy_url` if you're also configuring where not to proxy to).